sections/main-product.liquid

```diff
@@ -112,6 +112,16 @@
                   {{ product.description }}
                 </div>
               {%- endif -%}
+            {%- when 'sku' -%}
+              {%- assign current_variant = product.selected_or_first_available_variant -%}
+              <p
+                class="product__sku no-js-hidden{% if block.settings.text_style == 'uppercase' %} caption-with-letter-spacing{% elsif block.settings.text_style == 'subtitle' %} subtitle{% endif %}{% if current_variant.sku.size == 0 %} visibility-hidden{% endif %}"
+                id="Sku-{{ section.id }}"
+                role="status"
```

Comment:
I wonder if we really need to announce this at all. The price is critical for the buyer to make a decision to add it to cart or not, but I'm not sure if the SKU is as critical information to be announced every time the user switches the variant.

I say this as someone with little to no context on how that code is used outside of an inventory control context, though. Can you folks please share some thoughts on this decision and help me understand it a bit more? 👀

Reply:
@YoannJailin I was actually referring to announcing it to screen readers via the `role="status"` attribute every time the variant is changed. Is this what you mean when you said "dislpay"?

As much as we want assistive technologies users to get the same information, announcing too many things or things that are not as relevant to their experience can lead to a distracting UX for these users. That's why I was wondering if we really need to announce an 8-digit code when we update the variant.

Reply:
@eugenekasimov as previously discussed, I'm not sure how important the SKU is either. If it's important enough to display to sighted users, why would it be irrelevant to screen reader users? It's certainly not as important as the price, I agree with that. And we shouldn't announce too many things because that could get annoying. But, ideally, when the content on a page changes, those changes should be reflected to assistive tech users too.

I agree it shouldn't be a blocker. It's up to @YoannJailin and you to decide whether you want to leave it in.

Reply:
Okay, let me make this decision. For now let's keep it as is. We can discuss it and change later as a follow up.